SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -1016,65 +1018,67 @@ void FSpatialGDKEditorToolbarModule::StartInspectorProcess(TFunction<void()> OnR
 	const USpatialGDKEditorSettings* SpatialGDKEditorSettings = GetDefault<USpatialGDKEditorSettings>();
 	const FString InspectorVersion = SpatialGDKEditorSettings->GetInspectorVersion();
 	const uint16 RuntimeGRPCPort = SpatialGDKEditorSettings->GetDefaultReceptionistPort();
-	// If the Runtime port has changed, the previous Inspector process should be killed 
+	// If the Runtime port has changed, the previous Inspector process should be killed
```

Comment:
Just wanted to check I did not miss anything here... looks like just some reformatting.

Reply:
Yeah just formatting